pages/blog/[slug].vue

```diff
@@ -254,9 +268,11 @@ useHead({
     }
   }
 
-  // if there is no author content, hide the 'about the author' heading
-  :deep(.heading-about-author:not(:has(+.rich-text))) {
-    display: none;
+  .about-the-author {
+    @include ftva-subtitle-2;
+    color: #115daf;
+    /* $accent-blue; */
```

Comment:
@jendiamond Is the design token not working?

Reply:
the design token did not work here